nncf/common/graph/patterns/patterns.py

```diff
@@ -296,7 +307,7 @@ class HWFusedPatternNames(Enum):
     NORMALIZE_L2_MULTIPLY = PatternDesc("normalize_l2_multiply")
     SCALE_SHIFT = PatternDesc("scale_shift")
     SHIFT_SCALE = PatternDesc("shift_scale")
-    SE_BLOCK = PatternDesc("se_block")
+    SE_BLOCK = PatternDesc("se_block", ignored_algorithms=[AlgorithmType.NAS])
```

Comment:
Ignoring pattern for NAS looks good to me

Reply:
This is outdated now, I removed this functional, SE Block moved to ignored patters